### PR TITLE
Algorithm to sort an array of 0,1,2 in Julia

### PR DESCRIPTION
**Pull request by Saviour1001**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added the algorithm to sort an array of 0,1,2 without comparing the elements with each other in Julia
[REPLIT Link ](https://replit.com/@09HarshHarsh/012-Sort#main.jl)

### Checklist

- [ Yes ] I've read the contribution guidelines.
- [ Yes ] I've checked the issue list before deciding what to submit.
- [ Yes ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

This PR closes issue #6163

### CODE REVIEW

**Review by ankitaggarwal23 — COMMENTED** [review]
Read the instructions mentioned on Gssoc discord yesterday. Do the changes as per that.

**Comment by Saviour1001** [comment]
[REPLIT Link](https://replit.com/@09HarshHarsh/012-Sort#main.jl)

File: Julia/README.md [hunk 1]
```diff
@@ -67,6 +67,7 @@
 ## Sorting
 
 - [Bubble Sort](sort/bubble_sort.jl)
+- [0,1,2 Sort](sort/012_sort.jl)
```
Comment (ankitaggarwal23) [inline Julia/README.md:70]: use good naming conventions

File: Julia/sort/012_sort.jl [hunk 2]
```diff
@@ -0,0 +1,63 @@
+#=Sorting an array of 0,1,2 without comparing elements with each other or using any classic sorting 
+algorithm
+=#
```
Comment (ankitaggarwal23) [inline Julia/sort/012_sort.jl:3]: Add a good and understandable description

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**